### PR TITLE
Model database setup

### PR DESCRIPTION
**Pull request by PandaKao**
need to install postGIS to use database also redo database schema.sql

### CODE REVIEW

**Review by mirasoldavila13 — CHANGES_REQUESTED** [review]
Thank you for your hard work on this! I’ve reviewed the changes, and there are just a few adjustments needed before we can merge:

  1.  Revert changes to .github/ISSUE_TEMPLATE/user-story.md: We should keep this file consistent with our original template.

  2.  Remove client/package-lock.json, server/package-lock.json, and package-lock.json from the push: The versions here have fewer dependencies than what's in the main branch, and merging them could cause issues with our setup.

  3.  Uncomment server/config/connection.js: This file is necessary for Sequelize, and our deployment platform (Render) requires it to be in the config folder.

Once these changes are made, we’ll be ready to proceed with the merge. Let me know if you need any assistance!

**Comment by PandaKao** [comment]
Couldn't find where our user-story is, main doesn't have it. removed all package-locked.json as well as copied the package.json directly from main. Reinstated server/config/connection.js.

**Comment by mirasoldavila13** [comment]
@PandaKao I was checking out your index.js and noticed it's a bit different from what we have in the main branch. We're using Sequelize ORM to automatically load models and set up relationships, so we don't need to manually import them. It helps keep things flexible as we add more models.

Could you try pulling the latest from the main branch and adjust your file to match the dynamic setup we're using? Let me know if you need a hand with anything!

Thanks so much!

File: server/models/index.js
Comment (mirasoldavila13) [inline server/models/index.js:1]: @PandaKao  I was checking out your index.js and noticed it's a bit different from what we have in the main branch. We're using Sequelize ORM to automatically load models and set up relationships, so we don't need to manually import them. It helps keep things flexible as we add more models.

Could you try pulling the latest from the main branch and adjust your file to match the dynamic setup we're using? Let me know if you need a hand with anything! Here is the link : https://github.com/mirasoldavila13/NeighborhoodAid/blob/main/models/index.js

**Review by mirasoldavila13 — APPROVED** [review]
Looks great!

**Pull request merged**